src/system.rs

```diff
@@ -316,6 +316,7 @@ pub const MEMORY_MAPPED_IO: MemoryType = 0x0000000b;
 pub const MEMORY_MAPPED_IO_PORT_SPACE: MemoryType = 0x0000000c;
 pub const PAL_CODE: MemoryType = 0x0000000d;
 pub const PERSISTENT_MEMORY: MemoryType = 0x0000000e;
+pub const UNACCEPTED_MEMORY: MemoryType = 0x0000000f;
```

Comment:
The spec calls this `EfiUnacceptedMemoryType`. Only one other key uses the `Type` suffix, `EfiReservedMemoryType`, and there we preserved the suffix. We usually only strip parts of the name if it is used (mostly) consistently in the specification.

In this case, I modified the commit to use `UNACCEPTED_MEMORY_TYPE` (and amended the commit message accordingly). Let me know if you disagree.